### PR TITLE
Fix anchors and revise sentence

### LINKED ISSUE #25319: unexisting anchor in keydown_event
https://github.com/mdn/content/blob/de7e9d9937d2daab9a555e24fb498a09fdd7043d/files/en-us/web/api/element/keydown_event/index.md?plain=1#L74

```
[Keyboard locations](#keyboard_locations). 
```

is pointing to an unexisting anchor, that also are mentioning an unexisting section

### PR DESCRIPTION
**Pull request by dawei-wang**
Fix mdn/content#25319

### CODE REVIEW

File: files/en-us/web/api/element/keydown_event/index.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ _This interface also inherits properties of its parents, {{domxref("UIEvent")}}
     > **Note:** This does not describe the locale of the data being entered. A user may be using one keyboard layout while typing text in a different language.
 
 - {{domxref("KeyboardEvent.location")}} {{ReadOnlyInline}}
-  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](#keyboard_locations).
+  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEVent#keyboard_locations).
```
Comment (dawei-wang) [inline files/en-us/web/api/element/keydown_event/index.md:74]:
```suggestion
  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```

File: files/en-us/web/api/element/keyup_event/index.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ _This interface also inherits properties of its parents, {{domxref("UIEvent")}}
     > **Note:** This does not describe the locale of the data being entered. A user may be using one keyboard layout while typing text in a different language.
 
 - {{domxref("KeyboardEvent.location")}} {{ReadOnlyInline}}
-  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](#keyboard_locations).
+  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEVent#keyboard_locations).
```
Comment (dawei-wang) [inline files/en-us/web/api/element/keyup_event/index.md:72]:
```suggestion
  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```

File: files/en-us/web/api/element/keydown_event/index.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ _This interface also inherits properties of its parents, {{domxref("UIEvent")}}
     > **Note:** This does not describe the locale of the data being entered. A user may be using one keyboard layout while typing text in a different language.
 
 - {{domxref("KeyboardEvent.location")}} {{ReadOnlyInline}}
-  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](#keyboard_locations).
+  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```
Comment (dawei-wang) [inline files/en-us/web/api/element/keydown_event/index.md:74]:
```suggestion
  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```

File: files/en-us/web/api/element/keypress_event/index.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ _This interface also inherits properties of its parents, {{domxref("UIEvent")}}
     > **Note:** This does not describe the locale of the data being entered. A user may be using one keyboard layout while typing text in a different language.
 
 - {{domxref("KeyboardEvent.location")}} {{ReadOnlyInline}}
-  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](#keyboard_locations).
+  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```
Comment (dawei-wang) [inline files/en-us/web/api/element/keypress_event/index.md:63]:
```suggestion
  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```

File: files/en-us/web/api/element/keyup_event/index.md [hunk 5]
```diff
@@ -69,7 +69,7 @@ _This interface also inherits properties of its parents, {{domxref("UIEvent")}}
     > **Note:** This does not describe the locale of the data being entered. A user may be using one keyboard layout while typing text in a different language.
 
 - {{domxref("KeyboardEvent.location")}} {{ReadOnlyInline}}
-  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](#keyboard_locations).
+  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown above in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```
Comment (dawei-wang) [inline files/en-us/web/api/element/keyup_event/index.md:72]:
```suggestion
  - : Returns a number representing the location of the key on the keyboard or other input device. A list of the constants identifying the locations is shown in [Keyboard locations](/en-US/docs/Web/API/KeyboardEvent#keyboard_locations).
```

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you @dawei-wang !

**Pull request merged**